src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,93 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ clearInput,
+  addMovie,
+  findMovie,
+  searchedMovie,
+  found }) => {
+  const [query, setValue] = useState('');
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={ClassNames('input', { 'is-danger': !found })}
+              value={query}
+              onChange={(e) => {
+                setValue(e.target.value);
+                clearInput();
+              }}
+            />
+          </div>
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+          {found || (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={() => {
+                findMovie(query);
+              }}
+            >
+              Find a movie
+            </button>
+          </div>
+
+          <div className="control">
+            <button
+              type="button"
+              className="button is-primary"
+              onClick={() => {
+                addMovie();
+              }}
```

Comment:
pass just callback if no arguments are provided

```suggestion
              onClick={addMovie}
```